covid19-etl/etl/idph_facility.py

```diff
@@ -43,61 +43,34 @@ def files_to_submissions(self):
         if latest_submitted_date == today:
             print("Nothing to submit: today and latest submitted date are the same.")
             return
-        today_str = today.strftime("%Y%m%d")
-
-        print(f"Getting data for date: {today_str}")
-        url = "https://dph.illinois.gov/sitefiles/COVIDLTC.json"
-        self.parse_file(latest_submitted_date, url)
+        begin_date = (latest_submitted_date + datetime.timedelta(days=1)).strftime(
+            "%m/%d/%Y"
+        )
+        end_date = today.strftime("%m/%d/%Y")
+        print(f"Getting data for dates: {begin_date} - {end_date}")
+        url = f"https://idph.illinois.gov/DPHPublicInformation/api/COVIDExport/GetLTCFacilityHistorical?beginDate={begin_date}&endDate={end_date}"
+        self.parse_file(url)
 
-    def parse_file(self, latest_submitted_date, url):
+    def parse_file(self, url):
         """
         Converts a JSON files to data we can submit via Sheepdog. Stores the
         records to submit in `self.summary_locations` and `self.summary_clinicals`.
 
         Args:
-            latest_submitted_date (date): the date of latest available "summary_clinical" for project
             url (str): URL at which the JSON file is available
         """
         print("Getting data from {}".format(url))
         with closing(self.get(url, stream=True)) as r:
             data = r.json()
-            date = idph_get_date(data["LastUpdateDate"])
 
-            if latest_submitted_date and date == latest_submitted_date.strftime(
-                "%Y-%m-%d"
-            ):
+            if len(data) == 0:
                 print(
-                    "Nothing to submit: latest submitted date and date from data are the same."
+                    "Nothing to submit: No new data available in the given date range."
                 )
                 return
 
-            if "LTC_Reported_Cases" in data:
```

Comment:
any idea what this was?

Reply:
There was this key in the [previous source](https://dph.illinois.gov/sitefiles/COVIDLTC.json) which probably gave statewide cases and deaths information, which is no longer available in the [new source](https://idph.illinois.gov/DPHPublicInformation/api/COVIDExport/GetLTCFacilityHistorical)